src/utils/logic-app/designerUtils.ts

```diff
@@ -0,0 +1,633 @@
+
+                        const monacoLocale = getMonacoLocale($locale);
+
+                        if (monacoLocale !== "en") {
+                            global.require.config({
+                                "vs/nls": {
+                                    availableLanguages: {
+                                        "*": monacoLocale
+                                    }
+                                }
+                            });
+                        }
+
+                        const oauthService = new OAuth.OAuthPopupService("https://portal.azure.com");
+
+                        const getArmAccessToken = async () => {
+                            return "${authorization}".substring("Bearer ".length);
+                        };
+
+                        const getCurrentTenantId = async () => {
+                            // TODO?
+                        };
+
+                        const getCurrentUserObjectId = async () => {
+                            // TODO?
+                        };
+
+                        let designer;
+
+                        function initialize(options, analyticsContextData) {
+                            const config = {
+                                apiOperationsPath: \`/providers/Microsoft.Web/locations/\${options.location}/apiOperations\`,
+                                connectionsPath: "/providers/Microsoft.Web/connections",
+                                connectionProvidersPath: \`/providers/Microsoft.Web/locations/\${options.location}/managedApis\`,
+                                gatewaysPath: "/providers/Microsoft.Web/connectionGateways"
+                            };
+
+                            const urlService = new designercore.LogicAppsUrlService({
+                                baseUrl: options.baseUrl,
+                                config,
+                                integrationAccountId: options.integrationAccountId,
+                                location: options.location,
+                                resourceGroup: options.resourceGroup,
+                                subscriptionId: options.subscriptionId
+                            });
+
+                            const apiManagementServiceFactory = analytics => {
+                                return new designercore.LogicAppsApiManagementService({
+                                    analytics,
+                                    apiVersion: options.apiManagementApiVersion,
+                                    baseUrl: options.baseUrl,
+                                    getAccessToken: getArmAccessToken,
+                                    locale: $locale,
+                                    subscriptionId: options.subscriptionId
+                                });
+                            };
+
+                            const hostServiceFactory = () => ({
+                                canOpenAzureFunction: async resourceId => {
```

Comment:
Remove `canOpenAzureFunction` and `openAzureFunction` until they can be implemented.